public/locales/en/scholarshipForm.json

```diff
@@ -0,0 +1,25 @@
+{
+  "generalDescription": "Please add the general information about the scholarship.",
+  "scholarshipName": "Scholarship Name",
+  "organization": "Organization",
+  "scholarshipLink": "Scholarship Link",
+  "deadline": "Deadline",
+  "description": "Description",
+  "tags": "Tags",
+  "awardAmount": "Award Amount",
+  "requirementsDescription": "Include information that is required for applicants to have.",
+  "noEligibilityReqs": "NO ELIGIBILITY REQUIREMENTS",
+  "grades": "Grade(s)",
+  "gradesPlaceholder": "No grade requirements",
+  "minGpa": "Minimum GPA",
+  "schools": "School(s)",
+  "schoolsPlaceholder": "No school requirements",
+  "states": "State(s)",
+  "statesPlaceholder": "No state requirements",
+  "majors": "Major(s)",
+  "majorsPlaceholder": "No major requirements",
+  "ethnicity": "Ethnicity(s)",
+  "ethnicityPlaceholder": "No ethnicity requirements",
```

Comment:
Maybe for the empty placeholders we have a common string like "no requirements"?